datalad/cmd.py

```diff
@@ -169,26 +105,26 @@ def _log_nolog(self, *args):
         pass
 
     def _log_summary(self, fd, data):
-        fd_name = self.FD_NAMES[fd]
+        fd_name = self.fd_infos[fd]
         lgr.log(5, 'Read %i bytes from %i[%s]%s',
-                len(data), self.pid, fd_name, ':' if self._log_outputs else '')
+                len(data), self.process.pid, fd_name, ':' if self._log_outputs else '')
         if self._log_outputs:
             log_data = ensure_unicode(data)
             # The way we log is to stay consistent with Runner.
             # TODO: later we might just log in a single entry, without
             # fd_name prefix
             lgr.log(5, "%s| %s ", fd_name, log_data)
```

Comment:
Hard to say. The logger has to be enabled for level `5`, and the datalad configuration should set `datalad.log.outputs` to `True`. I do not see that in a quick glance over the tests.

Reply:
> The logger has to be enabled for level `5`, and the datalad configuration should set `datalad.log.outputs` to `True`. I do not see that in a quick glance over the tests.

https://github.com/datalad/datalad/blob/2fd46ee0dcd710ba99b1101e8172eace47dd8235/.travis.yml#L92-L100

Reply:
Thanks @kyleam Didn't look there! :-)

Reply:
I think `DATALAD_LOG_OUTPUTS=1` does not work because `1` is interpreted as `str`. It should be `yes` or similar to be interpreted as `True`. Changing it now, which should the execution of the hitherto untested lines.

Reply:
Fixed in a456f2bab1028127d7ed6616946e2683d3986db3. See issue #5678

Reply:
FWIW, changes highlighting says that it is covered now.  Might had been a fluke or smth fixed/extended.  I will consider this resolved.